### ISSUE TITLE
Switch to id-based translations?

### ISSUE THREAD

**Issue by monich** [Wunderfitz/harbour-fernschreiber#52]
Large number of duplicate strings is one problem with the current approach.

Another problem is the use of plurals - %1 members and such. In English and some other languages you may get away with something like (s) suffix but with some (e.g. Russian, Polish, Finnish) it's just plain impossible and phrases without proper plural case look incredibly stupid in such languages (it's certainly the case with Russian).

**Comment by Wunderfitz** [comment]
Yep, good point. The plural issue was already mentioned in #20. Will probably come eventually, but I will implement some additional features first.
